keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/unary/SigmoidVertex.java

```diff
@@ -0,0 +1,31 @@
+package io.improbable.keanu.vertices.dbl.nonprobabilistic.operators.unary;
+
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.diff.DualNumber;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.diff.Infinitesimal;
+
+public class SigmoidVertex extends DoubleUnaryOpVertex {
+
+    public SigmoidVertex(DoubleVertex inputVertex) {
+        super(inputVertex);
+    }
+
+    public SigmoidVertex(double inputValue) {
+        super(new ConstantDoubleVertex(inputValue));
+    }
+
+    @Override
+    protected Double op(Double x) {
+        return 1. / (1. + Math.exp(-x));
+    }
+
+    @Override
+    public DualNumber getDualNumber() {
```

Comment:
I've just merged in the `DualNumber` PR and messed this up for you - sorry!

Can you change this to `calculateDualNumber`, use `SinVertex` as an example. The maths will be the same.